data/extension/init/templates/tinygo/envoy.access_loggers/default/main.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
Can we include extension configuration into this init template ?

Reply:
Looks like the current template does not support vm configuration while plugin configuration is available. https://github.com/tetratelabs/getenvoy/issues/117

Meanwhile, I will add an stub of reading plugin configuration.

Reply:
And please note that there's a blocker in TinyGo on json package usage (https://github.com/tinygo-org/tinygo/pull/907), so for now we can only use gogoproto https://github.com/tinygo-org/tinygo/issues/447#issuecomment-617801438.

Reply:
Regarding no support for "vm configuration", it's a deliberate choice.

It's expensive to recreate a Wasm VM only to reconfigure an extension.

That is why extension developers should prefer extension configuration by default and fallback to "vm configuration" only when it is really necessary.

Having init templates to demonstrate how to use extension configuration is a better default choice.

It's not a goal to fit everything into init templates.

Reply:
I don't imply that should use JSON.

If a language like TinyGo or AssemblyScript doesn't have a ready-to-use JSON library, it's natural to choose a simpler configuration format (Envoy configuration model allows for any text or binary format).

E.g., you could choose `*.properties` format:
```
key1=value1
key2=value2
```
and read it into `map[string]string`

Reply:
OK, all makes sense. Thanks for clarifying. 

will work on a non-json simple format exmaple.